packages/template-chakra-storefront/src/pages/cart/partials/cart-modals.jsx

```diff
@@ -38,8 +38,7 @@ const CartModals = ({
     basket,
     handleUnavailableProducts
 }) => {
-    const modalProps = useDisclosure()
-
+    const confirmationModalProps = useDisclosure()
```

Comment:
Changing name to clarify which of these props are since CartModals handles 3 modals in this component